rakelib/security.rake

```diff
@@ -15,7 +15,7 @@ task :security do
   puts 'running bundle-audit to check for insecure dependencies...'
   exit!(1) unless ShellCommand.run('bundle-audit update')
   # TODO: remove --ignore CVE-2019-5420 after updating to rails 5.2
-  audit_result = ShellCommand.run('bundle-audit check --ignore CVE-2019-5420')
+  audit_result = ShellCommand.run('bundle-audit check')
```

Comment:
Thanks for doing this! Would love to see it as a separate commit

Reply:
Agreed this is good but for a different tiny-PR

Reply:
Can also remove the `#TODO` while you're at it

Reply:
Done ✔️